flutter/lib/ui/result_screen.dart

```diff
@@ -112,6 +112,8 @@ class _ResultScreenState extends State<ResultScreen>
     final pictureEdgeSize = 0.08 * MediaQuery.of(context).size.width;
 
     for (final benchmark in state.benchmarks) {
+      final backendDesc =
+          '${benchmark.benchmarkSetting.configuration} | ${benchmark.benchmarkSetting.acceleratorDesc}';
```

Comment:
Previous run could be made with different settings. For example, if someone reinstalls the app with different backend list, or with updated backend.
We should save the config info in the results and use it for showing previous results.

Reply:
Ok. I made the change to store the backend description in `_BriefResult` and `_FullResult`.
Because the `_BriefResult` is directly parsed from `RunResult` in `serializeBriefResults()`, I need to add `backendDescription` to both `RunResult` and `RunSettings`. I am not quite happy with this approach, since `backendDescription` is not used in the backend, but cannot find a cleaner approach. I am open to suggestion.

Reply:
We now have a use case for `backendDescription` in the backend part, see https://github.com/mlcommons/mobile_app_open/issues/203#issue-1097907108